eng/testing/tests.mobile.targets

```diff
@@ -14,7 +14,7 @@
     <!-- Suppress linker warnings as these are tests -->
     <SuppressTrimAnalysisWarnings>true</SuppressTrimAnalysisWarnings>
     <!-- Can be removed when SDK update picks up https://github.com/dotnet/sdk/pull/18655. -->
```

Comment:
This PR was merged two weeks ago.  Is the comment still accurate?

Reply:
No, the current plan is to continue to suppress warnings on this leg.  I removed the comment.